### DISCUSSION TITLE
Is it possible to have open yazi in a specific folder

### DISCUSSION THREAD

**Discussion by ITViking** [sxyazi/yazi#1948] [1. Q&A]
### What system are you running Yazi on?

Linux X11

### What terminal are you running Yazi in?

Bash with Zellij

### `yazi --debug` output

```Shell
none
```

### Describe the question

I'm using zellij and helix with the [yazi-filepicker](https://yazi-rs.github.io/docs/tips#helix-with-zellij). I want yazi to open a the folder that I'm currently in, is that possible? 

### Validations

- [X] I have searched the existing discussions/issues

**Comment by sxyazi** [comment]
Helix doesn't provide a way to get the current file being edited or the directory of the file being edited. 

So, since this isn't an issue with Yazi itself, unfortunately, it can't be solved on the Yazi side. It would need support from Helix. 

The good news is that there's an unmerged PR for it in Helix, https://github.com/helix-editor/helix/pull/11164

---

By the way, @luccahuguet has created a really cool project called [yazelix](https://github.com/luccahuguet/yazelix), which provides an alternative way to better integrate Yazi and Helix – using a sidebar instead of pop-ups. 

I think this solves your problem to some extent, because the sidebar is always there, so you don't have to worry about losing it.

**Comment by luccahuguet** [comment]
the relevant files are https://github.com/luccahuguet/yazelix/blob/main/yazi/open_file.nu

and https://github.com/luccahuguet/yazelix/blob/main/yazi/sidebar/yazi.toml

but of course you'd need zellij as well, and at this point you might as well just use yazelix
